### ISSUE TITLE
Javascript switch case indentation

### ISSUE THREAD

**Issue by jkevingutierrez** [rtts/djhtml#41]
Having an html like

```
<!DOCTYPE html>
<html>

<head>
    <title>Test</title>
</head>

<body>
    <script>
        const mode = 'approved'
        switch (mode) {
            case 'approved':
                console.log('Approved');
                break;
            case 'rejected':
                console.log('Rejected');
                break;
            default:
                console.log('Default');
        }
    </script>
</body>

</html>

```

Running djhtml is formatting the JS switch cases in a wrong way:

```
<!DOCTYPE html>
<html>

    <head>
        <title>Test</title>
    </head>

    <body>
        <script>
            const mode = 'approved'
            switch (mode) {
                case 'approved':
                console.log('Approved');
                break;
                case 'rejected':
                console.log('Rejected');
                break;
                default:
                console.log('Default');
            }
        </script>
    </body>

</html>
```

It is a dumb example, but djhtml is not formatting correctly the content inside each `case` block

Expected result should be

```
<!DOCTYPE html>
<html>

    <head>
        <title>Test</title>
    </head>

    <body>
        <script>
            const mode = 'approved'
            switch (mode) {
                case 'approved':
                    console.log('Approved');
                    break;
                case 'rejected':
                    console.log('Rejected');
                    break;
                default:
                    console.log('Default');
            }
        </script>
    </body>

</html>
```

In that case, the content inside the case statement should be indented.

**Comment by JaapJoris** [comment]
Thanks for the very clear and informative bug report!

You are right, currently there is no handling of Javascript `switch` statements, which means that everything inside one will be indented as if it were a regular `function() {}` block.

I promise this will be fixed in a future release of DjHTML! I'll keep you updated in this thread when that happens.

**Comment by JaapJoris** [comment]
My apologies for the long delay. First, I totally forgot about this, and then it turned out to be harder than expected. Stay tuned!

**Comment by JaapJoris** [comment]
To @jkevingutierrez and others following this: Great news! The just-released [DjHTML v1.4.11](https://github.com/rtts/djhtml/releases/tag/v1.4.11) now has proper support for the JavaScript `switch` statement! \o/

Big kudos to @pawelpel who picked this up where I left off. Thanks Paweł!
